### ISSUE TITLE
The web app doesn't refresh itself for feedback

### ISSUE THREAD

**Issue by cjunekim** [helgoboss/realearn#383]
When a control element's Label 1 position is Above top AND Label 1 sized is checked, the element is refreshed for the feedback automatically.

However, when the label 1's setting is other than the above specific setting, the element's visual isn't updated at all by itself. It is updated when another element's visual is updated -- if that one meets that specific condition.

**Comment by helgoboss** [comment]
Are you sure that's not related to a ReaLearn target which doesn't support automatic feedback? Please check first, thanks.

**Comment by cjunekim** [comment]
> Are you sure that's not related to a ReaLearn target which doesn't support automatic feedback? Please check first, thanks.

Yes, I am. Maybe it wasn't obvious from the screencast that I shared above. I haven't changed the target at all during my recording. The only thing that I changed was the Label 1 position and Label 1 sized checkbox. Initially, when they were default values, the elements update their states immediately as I move the midi controller. However, after changing the Label 1 position or Label 1 sized checkbox, that element won't update its visual at all by itself.

**Comment by faldurn** [comment]
This should clarify the precise conditions related to the bug.

My environment:
ReaLearn v2.12.0/x86_64 rev df8fc-dirty (2022-03-21 17:21:04 UTC)
Reaper v6.58 (win10)

I also tested this in ReaLearn v2.13.0-pre.4, with the same results.

The essence of this bug is that change to the feedback value(s) of circular projected control element(s) with horizontal text fails to trigger projection display updates.

1. The bug is independent of projection destination.
Projection via android tablet gives identical results to projection via web browser.
2. The bug only affects the projected control element's display.  Synchronization between associated physical controller(s) and target parameter(s) continues to operate correctly.
3. The update of any unaffected control element in a projection triggers a display update including an update of the affected control element(s) to the correct feedback value(s).
4. Only circular control elements where the text is either centered or not "Sized" are affected.  In other words, if a circular control element has text following the circular outline, display update will not be affected, whereas if the text is horizontal then display update will always be bugged.  Above/below top/bottom is irrelevant.

As an example, I had a correctly working projection involving several rectangular control elements representing motorized faders.
After changing 2 of these elements to circular with horizontal text, altering the feedback value either by moving the physical fader or by adjusting the target parameter failed to trigger a display update but the physical fader and target parameter continued to be synchronized.  Changing the feedback value of any control element with a rectangular display resulted in a display update, with the 2 affected control elements then updating to their current feedback value.

**Comment by mikekravets** [comment]
Hi! I'm stuck now with the same bug. 
Interesting, but when you choose square shape(For fader projection) and straight text - it works.
Round shape(For encoder projection) and straight text - does not work, updates position after turning other mapping parameter, which is currently working.
Would be cool to have encoders projection and straight text, it's much easier to read :)
![ezgif-4-1d3b15bbe9](https://github.com/helgoboss/realearn/assets/69674915/d92cbd8b-37a8-4ac2-8506-d4a4b4a48eab)
